brianlong/www.validators.app
# 184296810 - show stake pool logo

#### What's this PR do?
Place stake pool logos near by the validator identity.

#### How should this be manually tested?
- Test if pools list is assigned correctly. You need to have some StakeAccounts locally (gather_stake_accounts.rb script). Then run `rails r script/update_validator_stake_pools_list.rb`. Confirm that script runs without any errors and populates stake_pools_list attribute in validators table.
- Compare logos displayed on Validators index versus Stake Pools page - the logo on Validators index should be displayed only if stake is larger than 1 SOL (see screenshots below).
- Confirm that logos on Stake Pools page are displayed correctly (the first column in stake account row, and the logos in the card on top of the page)
- Confirm that there're no errors if no pools are assigned to validator (no logos)
- Confirm that logos on the homepage (in Stake Pools card) are displayed correctly and shuffled on page reload
- Confirm that stake pools tasks still work as expected, run `bundle exec rake add_stake_pools:mainnet` and `bundle exec rake update_fee_in_stake_pools:mainnet`

<img width="1079" alt="Zrzut ekranu 2023-03-24 o 15 39 08" src="https://user-images.githubusercontent.com/32059143/227556249-83ffd1da-7980-4a5f-9f61-480e4fec6a83.png">

<img width="1079" alt="Zrzut ekranu 2023-03-24 o 15 40 02" src="https://user-images.githubusercontent.com/32059143/227556260-5cead57f-687d-414e-aa01-a21e248a4dfd.png">

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/184296810)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
